src/plugins/lightscript.js

```diff
@@ -143,7 +202,7 @@ pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
   // must start with colon or arrow
   if (isExpression) {
     this.expect(tt.colon);
-    if (!this.isLineBreak()) return this.parseMaybeAssign();
+    if (!this.isLineBreak()) return this.parseWhiteBlockBody(node, indentLevel, true, false);
```

Comment:
Yeah, I think that's best handled elsewhere since they can happen within a block anyway:

```
x = if true:
  1 + 1
  break
  return
else:
  foo()
  continue
```
etc

though actually I think break and continue might be fine in there...